src/components/CompaniesGrid/style.module.scss

```diff
@@ -0,0 +1,45 @@
+@use "src/styles/vars.scss" as vars;
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(20rem, 1fr));
+  margin-top: 4rem;
+  row-gap: 1.7rem;
+  column-gap: 1.25rem;
+
+  .companyItem {
+    background: vars.$grey;
+    border-radius: 1.25rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2rem 2rem 0.75rem 2rem;
+    text-decoration: none;
+
+    &:hover {
+      transform: scale(1.03);
+      transition: 0.5s ease-in-out;
+    }
+    > .companyName {
+      margin: 0;
+      font-style: normal;
+      font-weight: 600;
+      font-size: 1.12rem;
+      line-height: 1.2rem;
+      color: #120e46;
+    }
+
+    .companyLogo {
+      height: 6rem;
+      overflow: hidden;
```

Comment:
I can't tell exactly just from looking at it but can we double check and confirm that we're preserving the image aspect ratio and not stretching any of the logos? See below image as reference

![Screenshot 2022-11-11 at 2 12 43 AM](https://user-images.githubusercontent.com/33165426/201318411-60c23edd-9575-4fc8-a733-07917c845c72.jpg)

![Screenshot 2022-11-11 at 2 12 57 AM](https://user-images.githubusercontent.com/33165426/201318459-96ebdcd1-9c2f-4cc3-b687-24e32b553839.jpg)